lib/static_sync/processor.rb

```diff
@@ -12,6 +12,7 @@ class ConflictError < StandardError
     def initialize(config, storage = nil)
       @config  = config
       @storage = storage || StaticSync::Storage.new(config)
+      @skip    = false
```

Comment:
Should `@skip` to be scoped by the file or the processor? The reason I ask is because `@skip` is set in `#handle_conflict` which seems to be scoped by files.